v2/pkg/operators/common/dsl/dsl.go

```diff
@@ -78,6 +78,26 @@ func init() {
 		"to_lower": makeDslFunction(1, func(args ...interface{}) (interface{}, error) {
 			return strings.ToLower(types.ToString(args[0])), nil
 		}),
+		"sort": makeDslWithOptionalArgsFunction(
+			"(args ...interface{}) interface{}",
+			func(args ...interface{}) (interface{}, error) {
+				argCount := len(args)
+				if argCount <= 0 {
+					return nil, invalidDslFunctionError
+				} else if argCount == 1 {
+					tokens := strings.Split(types.ToString(args[0]), "")
```

Comment:
I think we can use runes here and avoid the `split` + `join`:
```go
runes := []rune(types.ToString(args[0]))
sort.Slice(runes, func(i int, j int) bool {
    return runes[i] < runes[j]
})
return string(runes), nil
```

Reply:
Done.